setup.py

```diff
@@ -91,7 +91,7 @@
         "requests",
         "scipy",
         # for watertap.ui.api_model (though may be generally useful)
-        "pydantic < 2",
+        "pydantic >= 2",
```

Comment:
Just to be on the safe side

```suggestion
        "pydantic >= 2, <3",
```